File: frontend/lib/models/manager_profile_dto.dart
class ManagerProfileDto {
  final String fullName;
  final String? contactEmail;
  final String? contactPhone;
  final int? clubId;

  ManagerProfileDto({
    required this.fullName,
    this.contactEmail,
    this.contactPhone,
    this.clubId,
  });

  Map<String, dynamic> toJson() => {
        'fullName': fullName,
        'contactEmail': contactEmail,
        'contactPhone': contactPhone,
        'clubId': clubId,
      };

  factory ManagerProfileDto.fromJson(Map<String, dynamic> json) {
    return ManagerProfileDto(
      fullName: json['fullName'] as String,
      contactEmail: json['contactEmail'] as String?,
      contactPhone: json['contactPhone'] as String?,
      clubId: json['clubId'] as int?,
    );
  }
}

File: backend/src/main/java/ru/bowling/bowlingapp/DTO/ManagerProfileDTO.java
package ru.bowling.bowlingapp.DTO;

import jakarta.validation.constraints.Email;
import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.NotNull;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ManagerProfileDTO {
    @NotBlank
    private String fullName;

    @Email(message = "Invalid email format")
    private String contactEmail;

    private String contactPhone;

    @NotNull(message = "Club selection is required for manager profile")
    private Long clubId;
}
